### DISCUSSION TITLE
Turbo Repo + React Native w/o Expo

### DISCUSSION THREAD

**Discussion by ghost** [vercel/turborepo#2063] [Help] [answered]
I am really struggling to get React Native to work inside a my Turbo Repo Monorepo.  I am not trying to do anything complex, I am just trying to get the boilerplate React Native application for Android to run locally.  I am really struggling to find any sort of guide that does not use Expo

A small caveat for my project.

1. We are going through a private npm/maven registry, I think that I have everything configured correctly to point to private maven registry link.
2. We are not using Expo because frankly we were a little worried about it because nobody at our company is familiar with how it works and we cannot have our code/application being sent outside of our network and onto personal devices.

This project works just fine outside of the monorepo, everything installs and runs as expected, it seems to really only have these issues once inside the monorepo and the `node_modules` folder is in the root of the monorepo.  I did go through and update all of the paths to the node_modules folders that I could find.

I have seen this [example ](https://github.com/Enricopv/turbo-boilerplate)but I would prefer to use the new packages and I am not sure I want to just set this up without understanding what is happening.  I don't want to have to go in and strip everything out, I am looking to just get a simple POC running.

![image](https://user-images.githubusercontent.com/71139127/191989138-950c355f-fc3a-4e87-b953-d6d2fa89b4d7.png)

**Comment by AyoParadis** [comment]
did you figure this out?

**Comment by mehulkar** [comment]
@sgreco-dev could you share your example repo?

**Comment by anthonyshew** [comment, marked as answer]
Closing as stale. We've also recently updated the examples folder, including the `with-react-native` example!

**Reply by dev-BenYehuda** [reply]
This doesnt work
